packages/rum-core/src/domain/rumEventsCollection/view/trackTimings.ts

```diff
@@ -137,17 +145,19 @@ export function trackFirstInputTimings(
   const firstHidden = trackFirstHidden()
 
   const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, (entry) => {
-    if (
-      entry.entryType === 'first-input' &&
-      entry.startTime < firstHidden.timeStamp &&
+    if (entry.entryType === 'first-input' && entry.startTime < firstHidden.timeStamp) {
       // Discard invalid first-input entries, see
       // https://bugs.chromium.org/p/chromium/issues/detail?id=1185815
-      entry.startTime <= entry.processingStart
-    ) {
-      callback({
-        firstInputDelay: elapsed(entry.startTime, entry.processingStart),
-        firstInputTime: entry.startTime as Duration,
-      })
+      if (entry.startTime <= entry.processingStart) {
+        callback({
+          firstInputDelay: elapsed(entry.startTime, entry.processingStart),
+          firstInputTime: entry.startTime as Duration,
+        })
+      } else if (entry.toJSON) {
+        addMonitoringMessage(`Negative FID for entry ${entry.name}`, {
+          entry: { ...entry.toJSON(), target: undefined },
```

Comment:
could it be interesting to have the negative value in the log as well?

Reply:
So I don't think so, since we can compute it simply by doing `processingStart - startTime`, but I can add it if you want

Reply:
it could be more convenient if we want to quickly see the value or perform some query on it
I'd be in favor of adding it, just in case